Defs/ResearchProjectDefs/ResearchProjects_SAL.xml

```diff
@@ -111,10 +111,10 @@
   <!-- ==================== Speedy ============================= -->
   <ResearchProjectDef ParentName="PRF_SALResearch">
     <defName>PRF_SALResearchVI</defName>
-    <label>S.A.L Autonomous Overclocking I</label>
+    <label>S.A.L autonomous overclocking I</label>
     <techLevel>Spacer</techLevel>
     <baseCost>5000</baseCost>
-    <description>General S.A.L machines often work too slowly and can result in slow production. Upgrading them with the ability to overclock their operating systems, this will result in faster production speeds at the cost of some item quality.</description>
```

Comment:
I think maybe it will be better to remove comma here. "Upgrading them with the ability to overclock their operating systems will result in faster production speeds at the cost of some item quality."